REVIEW_COPY.md

```diff
@@ -0,0 +1,312 @@
+# Drag & Drop without Draggables & Dropzones
+
+Drag & drop UIs have been around for a while, and so you would be forgiven for thinking they're a solved problem. Certainly there's high quality libraries to help us build drag & drop UIs, and these days we even have an official HTML5 drag & drop API! Why then is building drag & drop UIs so hard?
+
+Having built some drag & drop UIs I've come to the conclusion a big part of the problem is our choice of tools. Those tools are the draggables and dropzones most libraries provide for building drag & drop interactions. Not using draggables and dropzones to build UIs often simplifies their construction massively.
+
+That's a pretty sweeping statement for a broad term like 'drag & drop', so I'd better back it up! To do so I'll show you three examples of drag & drop UIs built without draggables or dropzones. These examples come from real projects, but I've stripped them down to their drag & drop essentials for this post. I'll include what I believe to be the most relevant code snippets in those post, and invite you to look at [the full source code] if you'd like to learn more!
+
+Before we start with those examples, lets start with a quick refresher on draggables and dropzones.
+
+## A Refresher on Draggables & Dropzones
+
+To learn what draggables and dropzones are lets look at [Trello], an application where they are easy to identify. In Trello you can drag cards between lists. This makes cards draggables and lists dropzones.
+
+![A screenshot of the Trello app, showing some lists and cards][trello.png]
+
+On closer inspection though things aren't as clear cut. For example, when you drop a card next to a dropzone you might expect that card to return to its original list, or even get lost. But what happens is that the card will land in the list nearest to where we dropped it. That's much nicer for the user, but it's not clear how we might achieve it using draggables and dropzones.
```

Comment:
might be good to say what in the nature of a drop zone precludes this behavior. It's just that you're not dropping the draggle *on* the dropzone, right?

Reply:
Yeah, that's it. I've struggled a bit to make that point clear. Took another stab at it and reorganized the section to read:

> A draggable is a UI element that follows the cursor when the user presses down on it. A dropzone is a UI element that gets an event when we release a draggable over it.
> 
> For an example lets look at the application [Trello]. In Trello you organize cards in lists. We could make the cards as draggables and the lists dropzones. That way we'd get an event every time a card gets dropped on a list, which we could use to update our app's state with the new location of the card.
> 
> ![A screenshot of the Trello app, showing a card being dragged from one list to another.][trello.png]
> 
> On closer inspection though things aren't as clear cut. For example, in the real Trello application you don't need to drop a card on top of a list. If you drop the card on some empty space it will land in the list nearest to where we dropped it. That's much nicer for the user, but it's not clear how we might achieve it using draggables and dropzones.